addons/mail/static/src/models/time/time.js

```diff
@@ -0,0 +1,56 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one2one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'Time',
+    identifyingFields: ['messaging'],
+    lifecycleHooks: {
+        _created() {
+            // TODO evrything should be done in a single setInterval and the field should be updated by comparing value. this.day !== current day.
+
+            // Update the minute field every minutes
+            this.intervalMinute = setInterval(
+                () => this.update({ minute: new Date() }),
+                60 * 1000
+            );
+            // Update the hour field every hours
+            this.intervalHour = setInterval(
+                () => this.update({ minute: new Date() }),
+                60 * (60 * 1000)
+            );
+            // Update day every days
+            // TODO this is wrong, it must update when the day changes, not
+            // after 24 hours
+            this.intervalDay = setInterval(
+                () => this.update({ day: new Date() }),
+                24 * (60 * (60 * 1000))
+            );
+        },
+        _willDelete() {
+            clearInterval(this.intervalMinute);
+            clearInterval(this.intervalHour);
+            clearInterval(this.intervalDay);
+        }
+    },
+    modelMethods: {},
+    recordMethods: {},
+    fields: {
+        day: attr({
+            default: new Date(),
+        }),
+        hour: attr({
+            default: new Date(),
+        }),
+        messaging: one2one('Messaging', {
+            inverse: 'time',
+            required: true,
+            readonly: true,
+        }),
+        minute: attr({
+            default: new Date(),
+        }),
```

Comment:
Consequently from previous comments, replace fields `Time/day`, `Time/hour` and `Time/minute` by single field `Time/currentDateEveryMinute`